### DISCUSSION TITLE
What is the difference between `bun add <package>` and `bun install <package>`?

### DISCUSSION THREAD

**Discussion by matronator** [oven-sh/bun#2727] [General]
Are they the same or what?

**Comment by Jarred-Sumner** [comment]
It's supposed to be the same

`bun add` came first and that's because most of the CLI for `bun install` was inspired by yarn v1 since that was what I used before `bun install`

**Reply by matronator** [reply]
Thanks! :)

**Reply by nam20485** [reply]
>It's _supposed_ to be the same

_"In theory there is no difference between theory and reality, but in reality there is."_

Given your choice to qualify, I am curious what the salient difference(s) are? Or are they supposedly insignificant?

**Reply by nam20485** [reply]
To continue your analogy, essentially:

``` javascript
bun add <package> === yarn add <package> === npm install <package>  // add a new package dependency (to package.json)
```
```javascript
bun install === yarn install === npm install // install all declared dependencies (from package.json)
```

... no?
